src/values/boolean-value.js

```diff
@@ -0,0 +1,47 @@
+import FormValue from './form-value';
+import TextUtils from '../utils/text-utils';
+import NumberUtils from '../utils/number-utils';
+
+export default class BooleanValue extends FormValue {
+  constructor(element, booleanValue) {
+    super(element, booleanValue);
+
+    this.booleanValue = !!booleanValue;
+  }
+
+  format({useDisplayValue = false}) {
+    return this.booleanValue.toString;
+  }
+
+  get displayValue() {
+    return this.booleanValue.toString();
+  }
+
+  get length() {
+    return 1;
+  }
+
+  get isEmpty() {
+    return false;
+  }
+
+  get searchableValue() {
+    return this.booleanValue.toString();
+  }
+
+  get columnValue() {
+    return this.booleanValue;
+  }
+
+  get multipleValues() {
+    return null;
+  }
+
+  toJSON() {
+    return this.booleanValue;
+  }
+
+  isEqual(booleanValue) {
+    return this.booleanValues === booleanValue;
```

Comment:
this.booleanValue

Reply:
Not quite. If you are comparing a true to a false you want to return false.

Reply:
I think John was pointing out it should be `this.booleanValue` instead of `this.booleanValues` (note the `s` on the latter one). `this.booleanValues` is not defined.